node_mongo_tutorial/routes/router.js

```diff
@@ -0,0 +1,54 @@
+const express = require("express");
+const router = express.Router();
+const Article = require("../models/blogmodules");
+
+router.get("/", async (req, res) => {
+	const blogs = await Article.find();
+	res.json(blogs);
```

Comment:
would you re-format your responses and be consistent on it all over. 
eg: {status, message, data}